jekyll/_cci2/server-3-install-migration.adoc

```diff
@@ -10,10 +10,10 @@ version:
 :toc: macro
 :toc-title:
 
-Migrating from 2.19.x to 3.0 requires that you backup your 2.19 installation and then restore the data to the new Server 3.0 installation.
-It does require an already operating Server 3.0 installation. Depending on the size of your data stores, the migration
-can take anywhere from a few minutes to a few hours. We recommend using a staging environment before completing this
-process in a production environment. This will not only allow you to gain a better understanding of the migration process,
+Migrating from 2.19.x to 3.0 requires backing up your 2.19 instance data (Mongo, Postgres, and Vault) and than restoring that data in a waiting Server 3.0
+instance. If you run into trouble you are able to fallback to your 2.19 instance. It does require an already operating Server 3.0 installation. Depending 
```

Comment:
Might be worth calling out the 2.19.x application will be stopped; in the event they need to rollback they will need to start the application back up.

Reply:
@ryanwohara sounds like a great addition.

Reply:
I've added this in my change below, but maybe we need to call out how to restart 2.19 in this event?